lib/main_micro_alp.pm

```diff
@@ -16,11 +16,14 @@ use main_ltp_loader 'load_kernel_tests';
 use main_containers qw(load_container_tests is_container_test load_container_engine_test);
 use main_publiccloud qw(load_publiccloud_download_repos);
 use main_security qw(load_security_tests is_security_test);
-use testapi qw(check_var get_required_var get_var set_var);
+use testapi qw(check_var get_required_var get_var set_var record_info);
+use testapi;
```

Comment:
you should pick one but not both . or you just saying that you import ALL functions from `testapi` or you define certain list of functions which you want to import